### PR TITLE
docs: Add missing configurations for cap run

### PR DESCRIPTION
**Pull request by dtarnawsky**
These arguments were added but not documented.

### CODE REVIEW

**Comment by dtarnawsky** [comment]
Hey @jcesarmobile, I added these additional parameters. I had no idea these were available either. I'll need to do a bit of testing to see if they work.

File: docs/cli/commands/run.md [hunk 1]
```diff
@@ -22,5 +22,10 @@ npx cap run [options] <platform>
 - `--list`: Print a list of target devices available to the given platform
 - `--no-sync`: do not run the sync command
 - `--scheme <schemeName>`: set the scheme of the iOS project
+- `--configuration <name>`: Configuration name of the iOS Scheme
 - `--target <id>`: Run on a specific target device
+- `--live-reload`: Enable Live Reload
+- `--host <host>`: Live Reload by serving the web view on the specified host
+- `--port <port>`: Live Reload by serving the web view on the specified port
+- `--forwardPorts <port1,port2>`: Automatically run "adb reverse" for better live-reloading support
```
Comment (jcesarmobile) [inline docs/cli/commands/run.md:30]:
```suggestion
- `--forwardPorts <port1:port2>`: Automatically run "adb reverse" for better live-reloading support
```
It has to be with the `:`, this is sent to native-run same way it's introduced and native-run will complain if you use `,`

File: docs/cli/commands/run.md [hunk 2]
```diff
@@ -22,5 +22,10 @@ npx cap run [options] <platform>
 - `--list`: Print a list of target devices available to the given platform
 - `--no-sync`: do not run the sync command
 - `--scheme <schemeName>`: set the scheme of the iOS project
+- `--configuration <name>`: Configuration name of the iOS Scheme
 - `--target <id>`: Run on a specific target device
+- `--live-reload`: Enable Live Reload
+- `--host <host>`: Live Reload by serving the web view on the specified host
+- `--port <port>`: Live Reload by serving the web view on the specified port
```
Comment (jcesarmobile) [inline docs/cli/commands/run.md:29]: The descriptions are a bit confusing, Capacitor doesn't actually serve the app for live reload, the user is supposed to have a server that provides live reload features and start it, and then use `--live-reload` to make capacitor load from the host specified in `--host` and the port specified in `--port` and those values should match the live reload server host/port, but the description seems to indicate that Capacitor will be the one in charge of serving.

File: docs/cli/commands/run.md [hunk 3]
```diff
@@ -22,5 +22,10 @@ npx cap run [options] <platform>
 - `--list`: Print a list of target devices available to the given platform
 - `--no-sync`: do not run the sync command
 - `--scheme <schemeName>`: set the scheme of the iOS project
+- `--configuration <name>`: Configuration name of the iOS Scheme
 - `--target <id>`: Run on a specific target device
+- `--live-reload`: Enable Live Reload
```
Comment (jcesarmobile) [inline docs/cli/commands/run.md:27]: `npx cap run --help` shows this as `-l, --live-reload`, if you don't want to show it like that, you can add a new line for `-l` and say "shorthand for --live-reload" or something like that

**Review by jcesarmobile — CHANGES_REQUESTED** [review]

File: docs/cli/commands/run.md (hunk 2 above)
Comment (dtarnawsky) [reply docs/cli/commands/run.md]: I've updated this. Hopefully its a bit clearer.

File: docs/cli/commands/run.md (hunk 3 above)
Comment (dtarnawsky) [reply docs/cli/commands/run.md:27]: I've added `-l`.

**Review by jcesarmobile — APPROVED** [review]

**Review by theproducer — APPROVED** [review]

**Pull request merged**